docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,378 @@
+in OpenShift, and that your tenant has access to them. Your OpenStack cluster will also need the following requirements for each SR-IOV instance:
+
+- An instance from the RHOSP quota
+- Two ports from the RHOSP quota
+- A flavor with at least 16 GB memory, 4 vCPUs, and 25 GB storage space
+
+Additionally, for clusters that use single-root input/output virtualization (SR-IOV), RHOSP compute nodes require a flavor that supports [huge pages](huge-pages).
+Deploying worker nodes with SR-IOV networks is supported as a post install operation in the installer for both IPI and UPI workflows. Once you have verified that your OpenStack cluster can support SR-IOV in OpenShift, and have succesfully installed an OpenShift cluster that meets the [minimum requirements](README.md#openstack-requirements), you can follow these steps and example workflows to create worker nodes with SR-IOV NICs.
+
+## Preparing your OpenShift Cluster to use SR-IOV Networks
+
+Before you use single root I/O virtualization (SR-IOV) on a cluster that runs on Red Hat OpenStack Platform (RHOSP), make the RHOSP metadata service mountable as a drive and enable the No-IOMMU Operator for the virtual function I/O (VFIO) driver.
+
+### Enabling the RHOSP metadata service as a mountable drive
+
+Next, you will need to create a machine config that enables the SR-IOV operator to get the UUID of networks from OpenStack. This machine config will allow the OpenStack metadata service to be available as a mountable drive.
+
+```yaml
+kind: MachineConfig
+apiVersion: machineconfiguration.openshift.io/v1
+metadata:
+  name: 20-mount-config
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  osImageURL: ''
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+        - name: create-mountpoint-var-config.service
+          enabled: true
+          contents: |
+            [Unit]
+            Description=Create mountpoint /var/config
+            Before=kubelet.service
+
+            [Service]
+            ExecStart=/bin/mkdir -p /var/config
+
+            [Install]
+            WantedBy=var-config.mount
+
+        - name: var-config.mount
+          enabled: true
+          contents: |
+            [Unit]
+            Before=local-fs.target
+            [Mount]
+            Where=/var/config
+            What=/dev/disk/by-label/config-2
+            [Install]
+            WantedBy=local-fs.target
+```
+
+### Enabling the No-IOMMU feature for the RHOSP VFIO driver
+
+You can apply a machine config to your machine pool that enables the No-IOMMU feature for the Red Hat OpenStack Platform (RHOSP) virtual function I/O (VFIO) driver. The RHOSP vfio-pci driver requires this feature.
```

Comment:
"your worker machine pool" 
also drop the "you can"

Reply:
is the "vfio-pci" driver the same things as the "RHOSP VFIO driver"?
if so, the paragraph can be made into one. by just chanfing "for the Red Hat", to "required by the Red Hat"

Reply:
will need @atyronesmith 's wisdom to answer that

Reply:
Wisdom?  :).     The MachineConfig enables the noiommu functionality for the vfio-pci driver inside the worker VM.  This is in the openshift space, not in OpenStack.